src/views/Simulation/constants/nodeLabels.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+export const NODE_LABELS = {
```

Comment:
The "label" word is already used for the text we display next to the graph nodes. Can we rename this constant and the `container.label` property to something else? Maybe "node type"?

Reply:
IMO, we should change the name of the attribute too :slightly_smiling_face: 
But we can discuss it if you want